src/clj2el/cli.clj

```diff
@@ -0,0 +1,29 @@
+(ns clj2el.cli
+  (:require
+   [clj2el.api :refer [clj2el]]
+   [clojure.edn :as edn]
+   [clojure.pprint :refer [pprint]]
+   [clojure.string :as str]))
+
+(defn read-multiple-forms [s]
+  (edn/read-string (str "[" s "]")))
+
+(defn prn-str-multiple-forms [forms]
+  (str/join "\n" (map prn-str forms)))
+
+(defn pprint-str [form] (with-out-str (pprint form)))
+
+(defn pprint-str-multiple-forms [forms]
+  (str/join "\n" (map pprint-str forms)))
+
+(defn -main [& args]
+  (let [argset (set args)
+        multiple? (contains? argset "--multiple") ;; assuming we don't want to depend on babashka.cli
```

Comment:
You can change the main-opts in the `bbin/bin` arguments to `-x cljel.exec` and then make a function that recieves a map

Reply:
Huh, good point. I didn't think of that. Thanks!

Though seeing what you did with the web interface makes me question whether we really need those options. I think `(edamame.core/parse-string-all s {:all true})` is a good default, and I can't really defend the need more configuration.